src/Components/Components/ref/Microsoft.AspNetCore.Components.netstandard2.0.Manual.cs

```diff
@@ -44,4 +44,149 @@ namespace Microsoft.AspNetCore.Components.RenderTree
+        protected virtual string FormatValueAsString(T value) { throw null; }
+        public override System.Threading.Tasks.Task SetParametersAsync(Microsoft.AspNetCore.Components.ParameterCollection parameters) { throw null; }
+        protected abstract bool TryParseValueFromString(string value, out T result, out string validationErrorMessage);
+    }
+
+    public partial class InputDate<T> : Microsoft.AspNetCore.Components.Forms.InputBase<T>
+    {
+        public InputDate() { }
+        [Microsoft.AspNetCore.Components.ParameterAttribute]
+        public string ParsingErrorMessage { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } private set { throw null; }}
+        protected override void BuildRenderTree(Microsoft.AspNetCore.Components.RenderTree.RenderTreeBuilder builder) { }
+        protected override string FormatValueAsString(T value) { throw null; }
+        protected override bool TryParseValueFromString(string value, out T result, out string validationErrorMessage) { throw null; }
+    }
+
+    public partial class InputNumber<T> : Microsoft.AspNetCore.Components.Forms.InputBase<T>
+    {
+        public InputNumber() { }
+        [Microsoft.AspNetCore.Components.ParameterAttribute]
+        public string ParsingErrorMessage { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } private set { throw null; }}
+        protected override void BuildRenderTree(Microsoft.AspNetCore.Components.RenderTree.RenderTreeBuilder builder) { }
+        protected override bool TryParseValueFromString(string value, out T result, out string validationErrorMessage) { throw null; }
+    }
+
+    public partial class InputSelect<T> : Microsoft.AspNetCore.Components.Forms.InputBase<T>
+    {
+        public InputSelect() { }
+        [Microsoft.AspNetCore.Components.ParameterAttribute]
+        public Microsoft.AspNetCore.Components.RenderFragment ChildContent { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } private set { throw null; }}
+        protected override void BuildRenderTree(Microsoft.AspNetCore.Components.RenderTree.RenderTreeBuilder builder) { }
+        protected override bool TryParseValueFromString(string value, out T result, out string validationErrorMessage) { throw null; }
+    }
+
+    public partial class ValidationMessage<T> : Microsoft.AspNetCore.Components.ComponentBase, System.IDisposable
+    {
+        public ValidationMessage() { }
+        [Microsoft.AspNetCore.Components.ParameterAttribute]
+        public System.Linq.Expressions.Expression<System.Func<T>> For { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } private set { throw null; }}
+        protected override void BuildRenderTree(Microsoft.AspNetCore.Components.RenderTree.RenderTreeBuilder builder) { }
+        protected override void OnParametersSet() { }
+        void System.IDisposable.Dispose() { }
```

Comment:
Could you clarify why only some of the InputBase subclasses are here? Why InputNumber but not InputText?

Reply:
Those types don't have any parameters.

Reply:
Makes sense. Thanks.